includes/class-sst-blocks-integration.php

```diff
@@ -0,0 +1,230 @@
+<?php
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationInterface;
+
+/**
+ * Class for integrating with WooCommerce Blocks
+ */
+class SST_Blocks_Integration implements IntegrationInterface {
+
+	/**
+	 * The name of the integration.
+	 *
+	 * @return string
+	 */
+	public function get_name() {
+		return 'simple-sales-tax';
+	}
+
+	/**
+	 * When called invokes any initialization/setup for the integration.
+	 */
+	public function initialize() {
+		$this->register_frontend_scripts();
+		$this->register_editor_scripts();
+		$this->register_editor_styles();
+
+		add_filter(
+			'the_content',
+			array( $this, 'force_exemption_block' ),
+			1
+		);
+	}
+
+	/**
+	 * Returns an array of script handles to enqueue in the frontend context.
+	 *
+	 * @return string[]
+	 */
+	public function get_script_handles() {
+		return array( 'sst-tax-exemption-block-frontend' );
+	}
+
+	/**
+	 * Returns an array of script handles to enqueue in the editor context.
+	 *
+	 * @return string[]
+	 */
+	public function get_editor_script_handles() {
+		return array( 'sst-tax-exemption-block-editor' );
+	}
+
+	/**
+	 * An array of key, value pairs of data made available to the block on the client side.
+	 *
+	 * @return array
+	 */
+	public function get_script_data() {
+		$certificates = SST_Certificates::get_certificates_formatted();
+		$options      = array(
+			'none' => 'None',
+			'new'  => 'Add new certificate',
+		);
+
+		foreach ( $certificates as $cert ) {
+			$options[ $cert['CertificateID'] ] = $cert['Description'];
+		}
+
+		$selected = WC()->session
+			? WC()->session->get( 'sst_certificate_id', 'none' )
+			: 'none';
+
+		if ( sst_is_user_tax_exempt() && $certificates ) {
+			$selected = current( array_keys( $certificates ) );
+		}
```

Comment:
When the certificate is auto-selected for an exempt customer, does that actually result in tax being removed? Same Q for `class-sst-checkout.php`.

This may need fixing.

Reply:
The existing implementation was bugged. Fixed in 37b7520.